src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -26,7 +26,7 @@
     <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
       <File Include="@(LibrariesRuntimeFiles)" TargetPath="framework" />
       <!-- exclude shared libs as we only need static ones -->
-      <LibPackageExcludes Include="framework\%2A%2A\%2A$(LibSuffix)"/>
+      <LibPackageExcludes Include="framework\%2A%2A\%2ANative$(LibSuffix)"/>
```

Comment:
going to merge this to unblock the build but I think checking IsNative would be better, like we do here: https://github.com/dotnet/runtime/blob/d8208737f8b1ede2c6673a89769dc29fb7a7f6af/src/tests/tracing/eventcounter/runtimecounters.csproj#L26

Reply:
Ah, I didn't knew about it has this metadata. I'll give it a try.

Reply:
Turned out to be much cleaner https://github.com/dotnet/runtime/pull/109683